Detectors/ITSMFT/common/simulation/src/Point.cxx

```diff
@@ -10,40 +10,25 @@ ClassImp(AliceO2::ITSMFT::Point)
 using std::cout;
 using std::endl;
 using namespace AliceO2::ITSMFT;
-
-Point::Point() : FairMCPoint(),
-                 mTrackStatus(0),
-                 mTrackStatusStart(0),
-                 mShunt(0),
-                 mStartX(0.),
-                 mStartY(0.),
-                 mStartZ(0.),
-                 mStartTime(0.),
-                 mTotalEnergy(0.)
+using namespace AliceO2; //::Base;
+
+Point::Point(int trackID, unsigned short detID, TVector3 startPos, TVector3 endPos, TVector3 startMom,
+             double startE,double endTime, double eLoss, unsigned char startStatus, unsigned char endStatus)
```

Comment:
constructors for objects which will be created billions of times should be "offered" to the compiler as inlineable